### PR TITLE
Add twitch.tv adblocking

### PR DESCRIPTION
**Pull request by ryanbr**
Initial testing of twitch.tv blocking.

### CODE REVIEW

File: filter_lists/list_catalog.json [hunk 1]
```diff
@@ -1213,6 +1213,26 @@
             }
         ]
     },
+    {
+        "uuid": "529a3f3b-7eba-4351-b986-d176a82e7f5a",
+        "title": "Brave Twitch adblocking rules",
+        "desc": "Remmove Twitch.tv adverts",
```
Comment (ShivanKaul) [inline filter_lists/list_catalog.json:1219]:
```suggestion
        "desc": "Remove ads and trackers on twitch.tv",
```

**Comment by ryanbr** [comment]
Pems uploaded

File: filter_lists/list_catalog.json [hunk 2]
```diff
@@ -1213,6 +1213,26 @@
             }
         ]
     },
+    {
+        "uuid": "529a3f3b-7eba-4351-b986-d176a82e7f5a",
+        "title": "Brave Twitch adblocking rules",
+        "desc": "Remove ads and trackers on twitch.tv",
+        "langs": [],
+        "default_enabled": true,
```
Comment (ShivanKaul) [inline filter_lists/list_catalog.json:1221]: We don't want this default_enabled, we'll enable via Griffin.

File: filter_lists/list_catalog.json [hunk 3]
```diff
@@ -1213,6 +1213,25 @@
             }
         ]
     },
+    {
+        "uuid": "529a3f3b-7eba-4351-b986-d176a82e7f5a",
+        "title": "Brave Twitch adblocking rules",
```
Comment (ShivanKaul) [inline filter_lists/list_catalog.json:1218]:
```suggestion
        "title": "Brave Twitch Adblock Rules",
```

File: filter_lists/list_catalog.json [hunk 4]
```diff
@@ -1213,6 +1213,25 @@
             }
         ]
     },
+    {
+        "uuid": "529a3f3b-7eba-4351-b986-d176a82e7f5a",
```
Comment (ShivanKaul) [inline filter_lists/list_catalog.json:1217]: Should be all caps
```suggestion
        "uuid": "529A3F3B-7EBA-4351-B986-D176A82E7F5A",
```

**Review by ShivanKaul — APPROVED** [review]

**Pull request merged**